mis_builder_cash_flow/report/mis_cash_flow_views.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<odoo>
+
+    <record model="ir.ui.view" id="mis_cash_flow_tree_view">
+        <field name="name">mis.cash_flow.tree</field>
+        <field name="model">mis.cash_flow</field>
+        <field name="arch" type="xml">
+            <tree create="false" edit="false" delete="false">
+                <field name="company_id" groups="base.group_multi_company"/>
+                <field name="line_type"/>
+                <field name="date"/>
+                <field name="name"/>
+                <field name="account_id"/>
+                <field name="debit" sum="Total"/>
+                <field name="credit" sum="Total"/>
+                <button type="object" string="Open Journal Item / Forecast Line" icon="fa-external-link" name="action_open_related_line"/>
+            </tree>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="mis_cash_flow_form_view">
+        <field name="name">mis.cash_flow.form</field>
+        <field name="model">mis.cash_flow</field>
+        <field name="arch" type="xml">
+            <form create="false" edit="false" delete="false">
+                <group>
+                    <group>
+                        <field name="date"/>
+                        <field name="name"/>
+                        <field name="account_id"/>
+                        <field name="move_line_id"/>
+                    </group>
+                    <group>
+                        <field name="debit" sum="Total"/>
+                        <field name="credit" sum="Total"/>
+                        <field name="company_id" groups="base.group_multi_company"/>
+                        <field name="line_type"/>
+                    </group>
+                </group>
+            </form>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="mis_cash_flow_search_view">
+        <field name="name">mis.cash_flow.search</field>
+        <field name="model">mis.cash_flow</field>
+        <field name="arch" type="xml">
+            <search>
+                <field name="name"/>
+                <field name="date"/>
+                <separator/>
+                <field name="account_id"/>
+                <field name="company_id" groups="base.group_multi_company"/>
```

Comment:
in general improve readibility with one attribute per line

Reply:
Done